src/components/memory/view/memory.html

```diff
@@ -0,0 +1,27 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3 class="panel-title">Memory</h3>
+      </div>
+      <div class="panel-body">
+        <div class="form-horizontal" ng-if="!$ctrl.isMemoryDefined()">
+          <div class="form-group">
+            <label class="col-md-8 control-label">Memory management has not been defined. Click
+              <button ng-click = "$ctrl.createDefaultMemory()"> here</button> to define it</label>
```

Comment:
I guess it makes sense to keep it consistent with the stores/loaders. Long term maybe we should come up with a generic option for removing an element from the config, something like a red cross at the top right of the individual screens. We could then apply this to indexing etc as well, but obviously not a priority with all the other stuff we have on.